kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/rest/JbpmRestCustomIntegrationTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.integrationtests.jbpm.rest;
+
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.kie.server.api.rest.RestURI.DOCUMENT_URI;
+import static org.kie.server.api.rest.RestURI.build;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kie.server.api.marshalling.MarshallingFormat;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.integrationtests.config.TestConfig;
+import org.kie.server.integrationtests.shared.KieServerDeployer;
+import org.kie.server.integrationtests.shared.KieServerUtil;
+import org.kie.server.integrationtests.shared.basetests.RestOnlyBaseIntegrationTest;
+
+
+public class JbpmRestCustomIntegrationTest extends RestOnlyBaseIntegrationTest {
```

Comment:
Please change the name of the class to something more descriptive, e.g. `StackTraceIncludedIntegrationTest` or something which says that we test stack trace. Also this class doesn't have to test only jbpm endpoints, so jbpm reference is not needed.

Reply:
Thanks for the feedback! Class name changed